src/v2/components/PhotoModal.tsx

```
@@ -0,0 +1,214 @@
+    (e: React.KeyboardEvent) => {
+      if (e.key === 'Escape') {
+        onClose();
+      }
+    },
+    [onClose],
+  );
+
+  return (
+    <div
+      className="fixed inset-0 z-50 overflow-hidden bg-black/95 dark:bg-black/98"
+      role="dialog"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<dialog>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)